docs/setup/visualization.md

```diff
@@ -18,11 +18,9 @@ The visualization tool is available as a standalone desktop application. You can
           Download for Mac OS </a>  
 - <a href="https://github.com/KhiopsML/kv-electron/releases/download/v11.0.2/khiops-visualization_11.0.2_amd64.deb">
           Download for Ubuntu </a>
-- <a href="https://github.com/KhiopsML/kv-electron/releases/download/v11.0.2/khiops-visualization-11.0.2.x86_64.rpm">
```

Comment:
Why remove the CentOS version? the incompatibility for the Khiops Desktop app doesn't affect the Visualization App, does it?

Reply:
Yes, but what would be the use-case for : 1/ not having Khiops on the CentOS ; 2/ having the visualization on the CentOS ? If we keep it, we should further comment in this sense IMHO.

Reply:
> This is the case for MacOS: no binary for Khiops, but the Visualization application is available for those who want to analyze what they've learned with khiops-python (via Conda). My opinion is that the option is important as long as we do not provide the widget.

OK, I see your point. I will undo the removal.

Reply:
Done.